Bundle/MaintenanceJob/StartJob.php

```diff
@@ -55,6 +54,17 @@ public function execute(LoggerInterface $logger)
             }
         }
 
+        foreach (\Kwf_Util_Process::getRunningWebProcesses() as $process) {
+            if ($process['cmd'] !== 'symfony kwc_newsletter:send') continue;
+
+            if (preg_match("#--newsletterId=([0-9]+)#", $process['args'], $matches)) {
+                $newsletterId = $matches[1];
+                if (!array_key_exists($newsletterId, $this->procs)) $this->procs[$newsletterId] = array();
+
+                $this->procs[$newsletterId][] = $process['pid'];
```

Comment:
this->procs könnte jetzt $procs sein, oder?

Reply:
@nsams behoben